main.py

```diff
@@ -229,7 +218,7 @@ def escape_path(path):
 		short_anim_arg = ',short_animation' if settings.settings['short_anim'] else ''
 		hdd_lock_arg = ',locked' if settings.settings['hdd_locked'] else ''
 		sys_memory = settings.settings['sys_memory'].split(' ')[0]+'M'
-		accelerator_arg = Xqemu.generateAcceleratorArg(settings.settings['use_accelerator'])
+		kvm_irqchip_arg = ',kernel_irqchip=off' if sys.platform == 'linux2' else ''
```

Comment:
This only has to be added if we turn on CPU acceleration, so it should be grouped with my other request for respecting the UI choice.